src/_cffi_src/openssl/x509_vfy.py

```diff
@@ -28,20 +28,7 @@
 typedef ... X509_STORE;
 typedef ... X509_VERIFY_PARAM;
 
-typedef struct x509_store_ctx_st X509_STORE_CTX;
-struct x509_store_ctx_st {
-    X509_STORE *ctx;
-    int current_method;
-    X509 *cert;
-    Cryptography_STACK_OF_X509 *untrusted;
-    Cryptography_STACK_OF_X509_CRL *crls;
-    X509_VERIFY_PARAM *param;
-    void *other_ctx;
-    int (*verify)(X509_STORE_CTX *);
-    int (*verify_cb)(int, X509_STORE_CTX *);
-    int (*get_issuer)(X509 **, X509_STORE_CTX *, X509 *);
-    ...;
-};
+typedef ... X509_STORE_CTX;
```

Comment:
Split this into a seperate PR then; I can do it if you like.

On Mon, Jun 27, 2016 at 7:22 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/_cffi_src/openssl/x509_vfy.py
> <https://github.com/pyca/cryptography/pull/2826#discussion_r68673978>:
>
> > @@ -28,20 +28,7 @@
> >  typedef ... X509_STORE;
> >  typedef ... X509_VERIFY_PARAM;
> >
> > -typedef struct x509_store_ctx_st X509_STORE_CTX;
> > -struct x509_store_ctx_st {
> > -    X509_STORE *ctx;
> > -    int current_method;
> > -    X509 *cert;
> > -    Cryptography_STACK_OF_X509 *untrusted;
> > -    Cryptography_STACK_OF_X509_CRL *crls;
> > -    X509_VERIFY_PARAM *param;
> > -    void *other_ctx;
> > -    int (*verify)(X509_STORE_CTX *);
> > -    int (*verify_cb)(int, X509_STORE_CTX *);
> > -    int (*get_issuer)(X509 **, X509_STORE_CTX *, X509 *);
> > -    ...;
> > -};
> > +typedef ... X509_STORE_CTX;
>
> It looks like 16.0 doesn't use this (The X509StoreContext object appears
> to treat it as opaque)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/pyca/cryptography/pull/2826/files/d6e61b2c6b4b352f3b657c2d03a11ddcccada953#r68673978>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAADBOUinDLu69XrSlkfKZMvZcNfY4KFks5qQFsZgaJpZM4HwBgf>
> .
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: D1B3 ADC0 E023 8CA6